app/src/main/java/edu/mills/cs115/fruitthief/database/FruitTreeDao.kt

```diff
@@ -0,0 +1,98 @@
+package edu.mills.cs115.fruitthief.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+
+/**
+ * DAO Database containing two tables, fruit_info_table and tree_info_table.
```

Comment:
A DAO isn't a database. The database is defined elsewhere.

There's no reason for the client to know the names of the tables.